screenpy_selenium/actions/move_mouse.py

```diff
@@ -98,7 +98,8 @@ def _add_action_to_chain(self, the_actor: Actor, the_chain: ActionChains) -> Non
         """Private method to add this Action to the chain."""
         if self.target is not None and self.offset is not None:
             the_chain.move_to_element_with_offset(
-                self.target.found_by(the_actor), *self.offset
+                self.target.found_by(the_actor),
+                *self.offset,
```

Comment:
ruff rule COM812 should not be in the configuration anymore.  No need to add the comma here.

Reply:
`COMB812` is ignored; rerunning the lint does not change this back. Could there be a preference to list multiple arguments on separate lines when we need to linebreak for length? I haven't combed through the rules to find why this might be happening.